GNUmakefile

```
@@ -64,7 +64,12 @@ terraform build provider:
 mmv1:
 	cd mmv1;\
 		bundle; \
-		bundle exec compiler.rb -e terraform -o $(OUTPUT_PATH) -v $(VERSION) $(mmv1_compile);
+		if [ "$(VERSION)" = "ga" ]; then \
+			bundle exec compiler.rb -a -e terraform -o $(OUTPUT_PATH) -v ga --no-docs $(mmv1_compile); \
```

`-a` here & in the next line conflicts with `mmv1_compile`

```suggestion
			bundle exec compiler.rb -e terraform -o $(OUTPUT_PATH) -v ga --no-docs $(mmv1_compile); \
```

---

Attempting to run `make provider` results in:

```
bundler: failed to load command: compiler.rb (compiler.rb)
compiler.rb:101:in `<top (required)>': Cannot use -p/--products and -a/--all simultaneously (RuntimeError)
```

---

Nice catch !!